spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/ZuulProxyConfiguration.java

```diff
@@ -88,7 +88,9 @@ public DiscoveryClientRouteLocator routeLocator() {
 	public PreDecorationFilter preDecorationFilter(RouteLocator routeLocator) {
 		return new PreDecorationFilter(routeLocator,
 				this.zuulProperties.isAddProxyHeaders(),
-				this.zuulProperties.isRemoveSemicolonContent());
+				this.zuulProperties.isRemoveSemicolonContent(),
+				this.server.getServletPrefix(),
+				this.zuulProperties);
```

Comment:
What's the point in passing `zuulProperties` in here as well as `zuulProperties.isAddProxyHeaders()` etc? It's probably best if `PreDecorationFilter` remains independent of `ZuulProperties`, so please change it that way if possible.